### PR TITLE
Add Espruino and Bangle.js 2

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Creation IDs are used to digitally identify unique creations of many forms. They
 
 ## `0xBxxx_xxxx`
 * `0xB0B0_0000` [Unexpected Maker](https://github.com/unexpectedmaker)
+* `0xBA00_0000` [Espruino / BAngle.js](./creations/espruino.md)
```
Comment (dhalbert) [inline README.md:38]: Is the capital `A` deliberate or just a typo?

```suggestion
* `0xBA00_0000` [Espruino / Bangle.js](./creations/espruino.md)
```

**Review by dhalbert — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tannewt) [reply README.md:38]: I did it deliberately to imply how I can up with the creator id. I'm ok if you think it should be lowercase.

Comment (dhalbert) [reply README.md:38]: It just looked like maybe a typo. You could add `BA for BAngle` or something like that on the rest of the line.

Comment (tannewt) [reply README.md]: Nah, I'll just switch it. Committed.

**Review by dhalbert — APPROVED** [review]

**Pull request merged**